CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### Performance enhancements
 
 ### Bug fixes
+* `skbio.TreeNode.tips()` enables the `include_self` parameter if the node of interest happens to be a tip.
```

Comment:
When `tips()` is called on an actual tree, `self` is never returned, even if `include_self=True`.

Do you have recommendations on better wording?